pkg/testhelpers/helpers.go

```diff
@@ -23,3 +25,11 @@ func WithMockDb(t *testing.T, f func(*sql.DB, sqlmock.Sqlmock)) {
 		t.Errorf("there were unfulfilled expectations: %s", err)
 	}
 }
+
+func TestFile(t *testing.T, filename string, dat []byte) string {
+	t.Helper()
+	path := filepath.Join(t.TempDir(), filename)
+	err := os.WriteFile(path, dat, 0o600)
```

Comment:
Most likely doesn't make any difference in this case, but there's specialized function in std library for creating temporary files called `ioutil.TempFile`.